src/sdk/accountspace_nb.js

```diff
@@ -201,8 +196,8 @@ class AccountSpaceNB {
                 username: iam_user.name.unwrap().split(IAM_SPLIT_CHARACTERS)[0],
                 arn: iam_user.iam_arn,
                 // TODO: GAP Need to save created date
-                create_date: new Date(),
-                // TODO: GAP Miising password_last_used
+                create_date: Date.now(),
```

Comment:
Is it a gap that we can do in a separate PR for users (add also the `creation_date`).
Not in the scope of the PR, but for later.

Reply:
it's low priority and will pick it up if time permits.